README.md

```diff
@@ -30,3 +30,38 @@ In the interest of keeping the documentation simple, please write all docstring
 ### Testing - again please read!
 These tools are only valuable if we can rely on them to perform exactly as we expect. So, we need testing. Please create tests for every function that you make, as much as you are able. Guidance/examples here for the moment: https://github.com/GeoUtils/georaster/blob/master/test/test_georaster.py
 https://github.com/corteva/rioxarray/blob/master/test/integration/test_integration__io.py
+
+
+
+### Examples
+
+**Coregister a DEM to another DEM**
+```python
+import GeoUtils as gu
+
+reference_dem = "path/to/reference.tif"
+dem_to_be_aligned = "path/to/dem.tif"
+mask = "path/to/mask.shp"  # This is optional. Could for example be glacier outlines.
+
+aligned_dem, error = gu.coreg.coregister(reference_dem, dem_to_be_aligned, mask=mask)
```

Comment:
If the argument that is expected for mask is a shapefile, should the variable be named shp or shpfile instead? I would use 'mask' for a raster type. This could also be an option to take a raster mask as input.

Reply:
Good suggestion! I'll make it accept either a raster or a vector. I'll make another pull request once I'm finished.

Reply:
@adehecq, this inclusion is now in PR #16